operator.yaml

```diff
@@ -12,6 +12,9 @@ spec:
       labels:
         name: redis-enterprise-operator
     spec:
+      securityContext:
+        runAsUser: 1001
+        fsGroup: 1001
```

Comment:
Is this a must ? I see today it runs with some random ID,
and if this is a must -  I assume the Service Rigger also needs to run with the same securityContext OW it won't work

Reply:
ok